packages/admin/cms-admin/src/translation/AzureAITranslatorProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import { gql, useApolloClient } from "@apollo/client";
+import { ContentTranslationServiceProvider } from "@comet/admin";
+import React from "react";
+
+import { useContentScope } from "../contentScope/Provider";
+import { GQLTranslateQuery, GQLTranslateQueryVariables } from "./AzureAITranslatorProvider.generated";
+
+interface AzureAITranslatorProps extends Omit<React.ComponentProps<typeof ContentTranslationServiceProvider>, "enabled" | "translate"> {
+    enabled?: boolean;
+}
+
+export const AzureAITranslatorProvider: React.FunctionComponent<AzureAITranslatorProps> = ({
```

Comment:
Nit. AI should be cased Ai here (filename as well)

Reply:
a53e44842567e4c67db107a79418e3ae95139072